internal/repo/repo.go

```diff
@@ -117,6 +128,17 @@ func FindFilePatternDirs(root, pattern string) ([]string, error) {
 			return walkErr
 		}
 
+		for _, p := range ignore {
+			ptrn := filepath.Join(root, p)
```

Comment:
nit: just to have the same as above 😉 

```suggestion
			pattern := filepath.Join(root, p)
```

Reply:
This will shadow the function parameter and fail the lint test.